web-app/js/portal/filter/ui/FilterGroupPanel.js

```diff
@@ -46,6 +46,34 @@ Portal.filter.ui.FilterGroupPanel = Ext.extend(Portal.filter.ui.GroupPanel, {
 
     _filtersUpdated: function() {
 
+        var featureParams = this._getFeatureParams();
+        Ext.Ajax.request({
+            url: "layer/getLayerInfoJson",
+            params: featureParams,
+            scope: this,
+            success: this._handleGetFeatureRequestMethod
+        });
+    },
+
+    _handleGetFeatureRequestMethod: function(results) {
+
+        (JSON.parse(results.responseText).owsType == "WCS") ?  this.getWcsFeatureCount(): this.getWfsFeatureCount();
+    },
+
+    getWcsFeatureCount: function() {
```

Comment:
Were not really getting counts here - would it be clearer if we made these methods return booleans/renamed them to reflect what they are actually doing - checking if there is any data for requested filters

Reply:
True. Changes now pushed @jonescc